### PR TITLE
Adds xeno heads to be used with xeno hybrids and xenolike races

### PR DESCRIPTION
**Pull request by kcin2001**
if the ears dmi hasnt been updated in the last few days it shouldnt have any problems

colorable xeno heads arent added in but the sprites are there for when i figure it out since part of it needs to be uncolored

### CODE REVIEW

**Comment by kcin2001** [comment]
as a note it will probably not with any snouted species

File: code/modules/vore/appearance/sprite_accessories_vr.dm [hunk 1]
```diff
@@ -221,6 +221,82 @@
...
+/datum/sprite_accessory/ears/xeno_head_drone_color
+	name = "colorable drone dome"
+	icon_state = "cxeno_drone"
+	gender = NEUTER
+/*		
+/datum/sprite_accessory/ears/xeno_head_sentinel_color
+	name = "colorable sentinal dome"
+	icon_state = "cxeno_sentinel"
+	gender = NEUTER
+*/
```
Comment (Cameron653) [inline code/modules/vore/appearance/sprite_accessories_vr.dm]: The comment above makes me think this commenting out should end at line 299 instead of 284.

**Comment by kcin2001** [comment]
actually all the others should be fine for coloring, the only one i need to figure out is the sentinal but you did remind me i forgot to add in the line to make them colorable

**Comment by kcin2001** [comment]
will this ever be merged?

**Comment by SpadesNeil** [comment]
tumblr_lhb8kgh5hv1qzz55x

**Comment by kcin2001** [comment]
you are too perfect of a meme

**Comment by SpadesNeil** [comment]
Can we have screenshots of the new heads? That's why I keep passing this over and somehow I haven't mentioned this yet.

**Comment by kcin2001** [comment]
Oh sorry sure, ill post a pic of the heads tommorow afternoon

**Comment by kcin2001** [comment]
Its more or less the same as the suggestion for the heads in the issues

**Comment by kcin2001** [comment]
showcase
@SpadesNeil here is the heads

**Comment by SpadesNeil** [comment]
Last question. These heads are only available for the Xeno-hybrid _race_, right? I do not see code that says that so I am not sure.

I will not allow these for regular crew because of how stupid the crew behaves when they see Xeno heads on things. They tend to murderbone it and although there's rules against it, it's unfair to get angry at the crew for murderboning something that usually tries to kill them.

**Comment by kcin2001** [comment]
To make them race restricted we need a way to make ears restrictable unless someone can help me with making them hair

**Comment by kcin2001** [comment]
Otherwise, whitelist

**Comment by Cameron653** [comment]
I think the xeno heads being available to everyone is fine, seeing as we also have xeno tails available to everyone ~~And this might actually get me to play my xeno-hybrid again~~

**Comment by SpadesNeil** [comment]
I'd like to keep it restricted to xeno hybrids.

**Comment by kcin2001** [comment]
Cameron i had that in prelim testing and it didnt do anything

**Comment by kcin2001** [comment]
I wasnt even admin so i cant even say it might have been that

**Comment by Cameron653** [comment]
...Huh. You're right. 
For some reason it just doesn't work on ears. Just tested and can confirm
http://i.imgur.com/qeUngCR.png

**Comment by kcin2001** [comment]
im guessing a line of code may make it work for ears but i dont know what

**Comment by Cameron653** [comment]
The alternate of adding it to hair (https://github.com/VOREStation/VOREStation/blob/748d97305f6a1def60b79c3fb2790b960c0d1466/code/modules/mob/new_player/sprite_accessories_vr.dm#L7)
would probably be best.
https://pastebin.com/raw/C3Ckzy5H (Copy and paste the code into sprite_accessories_vr.dm under /datum/sprite_accessory/hair)
And add the image files to icons/mob/human_face_vr.dmi

**Comment by kcin2001** [comment]
ill try doing hair again when i get home, how do you you undo commits btw?

**Comment by Cameron653** [comment]
I have no idea how to undo commits. Doesn't matter too much, honestly. You could either chose this PR and open a new one, or just not worry about the previous commits showing.

**Comment by kcin2001** [comment]
closing the pr doesnt make commits not show but i was more asking for ease of undoing the dmi change

**Comment by kcin2001** [comment]
ill just dl the lasest of that dmi and change it to that dmi

**Comment by SpadesNeil** [comment]
Sounds like we should mark "Do Not Merge" for now.

**Comment by kcin2001** [comment]
@SpadesNeil the pr should all be ready once travis is done

**Comment by kcin2001** [comment]
I also fixed the coloration on inkling hair (ears) since i was in the file and cameron pointed it out

**Comment by Cameron653** [comment]
Code looks good now. Merge it so it gets in the release, yeh?

**Comment by kcin2001** [comment]
When was the last time our hair was updated btw?

**Comment by Cameron653** [comment]
18 days ago.

https://github.com/VOREStation/VOREStation/commit/e59bb743c7b29add32ee6c3794edebdf87959f73

**Comment by kcin2001** [comment]
ill do dl the latest one and add mine just to be safe

**Comment by Cameron653** [comment]
Any more requests on this? If not, going to merge this soon so it gets included in the next Master -> Release

**Pull request merged**